chore: added the closing tag for dependencies

viveknaskar opened the pull request:
I found that the maven dependency for the google-cloud-bigquery was incomplete while I was revisiting the code. Hence, added the required closing tag to fix the issue.

viveknaskar commented:
Are there any changes required from my end for this PR to be merged? Please let me know!

stephaniewang526 commented:
Thank you for contributing. Could you update your initial commit to: `chore: added the closing tag for dependencies`.

Thanks!

viveknaskar commented:
> Thank you for contributing. Could you update your initial commit to: `chore: added the closing tag for dependencies`.
> 
> Thanks!

Updated the commit.

stephaniewang526 commented:
Your initial commiet is still `fix: added the closing tag for dependencies`. Please open a new PR to address this with the correct format.
